kakarot_scripts/constants.py

```diff
@@ -242,6 +242,18 @@ class ChainId(IntEnum):
 DATA_DIR = Path("kakarot_scripts") / "data"
 
 
+def load_pre_eip155_transactions():
+    raw_data = json.loads((DATA_DIR / "pre_eip155_txs.json").read_text())
+
+    for tx in raw_data.values():
+        tx["signed_tx"] = bytes.fromhex(tx["signed_tx"])
+
+    return raw_data
+
+
+PRE_EIP155_TX = load_pre_eip155_transactions()
```

Comment:
maybe a good opportunity to introduce pydantic, which would be something like 

```
class Pre155TransactionsModel(BaseModel):
    signed_tx: bytes

    @field_validator("signed_tx", mode="before")
    def to_bytes(v):
        bytes.fromhex(...)

PRE_EIP155_TX = Pre155TransactionsModel.model_validate(json.loads((DATA_DIR / "pre_eip155_txs.json").read_text()))
```

Reply:
another options is to just have the `pre_eip155_txs.json` a `pre_eip155_txs.py` with directly the right types